lib/couchdb.js

```diff
@@ -15,11 +15,15 @@ var url = require('url');
 var util = require('util');
 var _ = require('lodash');
 
+const DEFAULT_MODEL_VIEW = 'loopback__model__name';
+const DEFAULT_MODEL_PREFIX = 'LBModel';
+const DEFAULT_PROPERTY_PREFIX = 'LBIndex';
```

Comment:
I had some discussion with the team regarding this, see comment https://github.com/strongloop/loopback-connector-couchdb2/pull/10#issuecomment-321154953 and https://github.com/strongloop/loopback-connector-couchdb2/pull/10#discussion_r132090514

So tbh I am a little sick to allow the customized prefixes, and it we do want to support it, I think it would be good to have model & datasource level control.